icu4c/source/common/unicode/ucurr.h

```diff
@@ -113,7 +113,28 @@ typedef enum UCurrNameStyle {
      *
      * @stable ICU 61
      */
-    UCURR_NARROW_SYMBOL_NAME
+    UCURR_NARROW_SYMBOL_NAME,
+
+    /**
+     * Selector for getName() indicating the formal currency symbol.
+     * The formal currency symbol is similar to the regular currency
+     * symbol, but it always takes the form used in formal settings
+     * such as banking; for example, "NT$" instead of "$" for TWD in zh-TW.
+     *
+     * @draft ICU 67
+     * @provisional This API might change or be removed in a future release.
```

Comment:
Don't use the provisional boilerplate in ICU4C

Reply:
Removed `@provisional`